tests/scripts/__init__.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2020 Greenbone Networks GmbH
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from unittest.mock import patch, MagicMock, create_autospec
+from typing import Union
+from pathlib import Path
+from importlib.util import spec_from_file_location, module_from_spec
+
+from lxml import etree
+from gvm.errors import GvmResponseError
+from gvm.protocols.latest import Gmp
+
+# hacky ... hacky ... loading a script ...
+def load_script(path: Union[str, Path], script_name: str):
+    spec = spec_from_file_location(
+        script_name, '{}/{}.gmp.py'.format(path, script_name)
+    )
+    script = module_from_spec(spec)
+    spec.loader.exec_module(script)
+
+    return script
```

Comment:
I don't think this is hacky. You could also look at https://github.com/greenbone/gvm-tools/blob/e04cda21b7f3065ea79c95140aef051b86f99184/gvmtools/helper.py#L132 how the scripts are actually executed.

Reply:
But I think we are fine with this function to load the scripts for the tests?

Reply:
Of course I just find the comment misleading